frontend/src/src/components/UserProfile/UserProfile.css

```diff
@@ -0,0 +1,145 @@
+/* UserProfile.css */
+.user-profile {
+    margin: 0 auto;
+    padding: 20px;
```

Comment:
we should look at using rem instead of pixels

Reply:
cool, made the changes